homeassistant/components/esphome/climate.py

```diff
@@ -164,11 +164,17 @@ def _on_static_info_update(self, static_info: EntityInfo) -> None:
         )
         self._attr_min_temp = static_info.visual_min_temperature
         self._attr_max_temp = static_info.visual_max_temperature
+        self._attr_min_humidity = round(static_info.visual_min_humidity)
+        self._attr_max_humidity = round(static_info.visual_max_humidity)
         features = ClimateEntityFeature(0)
         if self._static_info.supports_two_point_target_temperature:
             features |= ClimateEntityFeature.TARGET_TEMPERATURE_RANGE
         else:
             features |= ClimateEntityFeature.TARGET_TEMPERATURE
+        if self._static_info.supports_target_humidity:
+            features |= ClimateEntityFeature.TARGET_HUMIDITY
+        if self._static_info.supports_aux_heat:
+            features |= ClimateEntityFeature.AUX_HEAT
```

Comment:
Aux heat had been deprecated. Should we revert this PR?

/CC @jesserockz @bdraco